packages/alfa-rules/test/sia-r14/rule.spec.tsx

```diff
@@ -45,6 +45,21 @@ test(`evaluate() passes a <button> element whose perceivable text content includ
   ]);
 });
 
+test(`evaluate() passes a <button> element whose perceivable text content includes symbols and
+      control characters and is included in its accessible name set by aria-label`, async (t) => {
+  const target = <button aria-label="Hello +-|­▼ world">Hello­←</button>;
```

Comment:
```suggestion
  const target = <button aria-label="Hello ← world">Hello­ +-|­▼</button>;
```
Since the rule check that the label is included in the name, I think it's a better test to have more garbage in the label (i.e `<button aria-label="Hello +-|­▼ world">Hello­</button>` would already pass the current version of the rule)